components/BottomNavigation/BUILD

```diff
@@ -51,30 +52,16 @@ objc_bundle(
     bundle_imports = [":BundleFiles"],
 )
 
-mdc_objc_library(
+mdc_extension_objc_library(
     name = "ColorThemer",
-    srcs = native.glob(["src/ColorThemer/*.m"]),
-    hdrs = native.glob(["src/ColorThemer/*.h"]),
-    includes = ["src/ColorThemer"],
-    sdk_frameworks = [
-        "UIKit",
-    ],
-    visibility = ["//visibility:public"],
     deps = [
         ":BottomNavigation",
         "//components/schemes/Color",
     ],
 )
 
-mdc_objc_library(
+mdc_extension_objc_library(
     name = "TypographyThemer",
-    srcs = native.glob(["src/TypographyThemer/*.m"]),
-    hdrs = native.glob(["src/TypographyThemer/*.h"]),
-    includes = ["src/TypographyThemer"],
-    sdk_frameworks = [
-        "Foundation",
-    ],
```

Comment:
I think we may need to preserve the sdk_frameworks here and throughout.

Reply:
Though it looks like we're being wonderfully inconsistent, so perhaps they're not required?

Reply:
I'd prefer we don't add these since it's redundant. By default targets for iOS, tvOS, and watchOS will include Foundation and UIKit.

https://docs.bazel.build/versions/master/be/objective-c.html

Reply:
Awesome!